doc/how_to/observers/ipywidget.md

```diff
@@ -0,0 +1,186 @@
+    center = param.List(default=[52.204793, 360.121558])
+    zoom = param.Number(default=4, bounds=(0, 24), step=1)
+
+parameterized = Map()
+
+pn.observers.ipywidget.sync_with_parameterized(
+    model=leaflet_map, parameterized=parameterized
+)
+pn.Column(leaflet_map, parameterized.param.zoom, parameterized.param.center).servable()
+```
+
+The `sync_with_parameterized` function synchronizes the shared traits/parameters `center` and `zoom` between the `leaflet_map` and `parameterized` objects.
+
+To specify a subset of traits/parameters to synchronize, use the `names` argument:
+
+```python
+pn.observers.ipywidget.sync_with_parameterized(
+    model=leaflet_map, parameterized=parameterized, names=("center",)
+)
+```
+
+The `names` argument can also be a dictionary mapping trait names to parameter names:
+
+```pyodide
+import panel as pn
+import ipyleaflet as ipyl
+import param
+
+pn.extension("ipywidgets")
+
+leaflet_map = ipyl.Map()
+
+class Map(param.Parameterized):
+    zoom_level = param.Number(default=4, bounds=(0, 24), step=1)
+
+parameterized = Map()
+
+pn.observers.ipywidget.sync_with_parameterized(
+    model=leaflet_map, parameterized=parameterized, names={"zoom": "zoom_level"}
+)
+pn.Column(leaflet_map, parameterized.param.zoom_level).servable()
+```
+
+## How to Create a Viewer from an ipywidget Instance
+
+To create a `Viewer` object from a ipywidget instance, use the `ModelViewer` class:
+
+```pyodide
+import panel as pn
+import ipyleaflet as ipyl
+
+pn.extension("ipywidgets")
+
+leaflet_map = ipyl.Map(zoom=4)
+
+viewer = pn.observers.ipywidget.ModelViewer(model=leaflet_map, sizing_mode="stretch_both")
+pn.Row(pn.Column(viewer.param, scroll=True), viewer, height=400).servable()
+```
+
+Check out the parameters to the left of the map, there you will find all the traits of the `leaflet_map` instance. Try changing some.
+
+To specify a subset of traits/parameters to synchronize, use the `_names` argument:
+
+```python
+viewer = pn.observers.ipywidget.ModelViewer(
+    model=leaflet_map, _names=("center",), sizing_mode="stretch_both"
+)
+```
+
+The `names` argument can also be a dictionary mapping trait names to parameter names.
+
+## How to Create a Viewer from an ipywidget Class
+
+To create a `Viewer` class from an ipywidget class, use the `ModelViewer` class:
+
+```pyodide
+import panel as pn
+import ipyleaflet as ipyl
+import param
+
+pn.extension("ipywidgets")
+
+class MapViewer(pn.observers.ipywidget.ModelViewer):
+    _model_class = ipyl.Map
+    _names = ["center", "zoom"]
+
+    zoom = param.Number(4, bounds=(0, 24), step=1)
+
+viewer = MapViewer(sizing_mode="stretch_both")
+
+pn.Row(pn.Column(viewer.param, scroll=True), viewer, height=400).servable()
+```
+
+The `_names` attribute is an optional iterable or dictionary. It specifies which traits to synchronize to which parameters.
+
+## How to Create a Reactive Value from the Trait of an ipywidget
+
+Use `create_rx` to create a reactive value from the trait of an ipywidget.
```

Comment:
Its quite constraining that its not possible to create bidirectionally synced widgets from reactive values. Do you know how to solve https://github.com/holoviz/panel/issues/6911 @philippjfr ?

Reply:
Apparently its not possible